### DISCUSSION TITLE
Berry sunset/sunrise

### DISCUSSION THREAD

**Discussion by RedBlackThunder** [arendst/Tasmota#15000] [Q&A] [answered]
Hi,

as I am started working with berry, is there any function or method to access sunrise and sunset data of tasmota from within berry? I haven't found anything in the documentation.

Thx,
Flo

**Comment by sfromis** [comment, marked as answer]
You can access lots of things via commands:
`var statustim=tasmota.cmd('Status 7')['StatusTIM']`
`print(statustim['Sunrise'], statustim['Sunset'])`
